README.adoc

```diff
@@ -191,7 +195,23 @@ RESOURCE_CPU_WEIGHT_pn-aktualizr = "50"
 
 == Development configuration
 
-There are a few settings that can be controlled in `local.conf` to simplify the development process:
+=== Logging
+
+During development, it is recommended to enable persistent systemd logging in case you run into problems. To do so, add this to your `local.conf`:
+
+....
+IMAGE_INSTALL_append += " systemd-journald-persistent"
```

Comment:
Note that this is enabled by default for newly configured builds: https://github.com/advancedtelematic/meta-updater/blob/7530d93ebb97b519a12b88d121320ac0bb711c46/conf/local.conf.sample.append#L26. Maybe the phrasing could mention it?

Reply:
Done. Also added a reference to historical support for the porter board.